logging/logger.go

```diff
@@ -1,8 +1,13 @@
 package logging
 
 import (
+	"io/ioutil"
 	"log"
 	"os"
 )
 
+func init() {
+	Infolog.SetOutput(ioutil.Discard)
+}
+
```

Comment:
Do we want to keep this? As far as I understand we discard all logs here?

Reply:
This was just moved from a different place. Kept it since I'm not 100% sure of the purpose of this. I think we can remove it, but I suggest not to include it in this PR since it's unrelated